libraries/AP_HAL_ChibiOS/hwdef/YJUAV_A6Ultra/hwdef.dat

```diff
@@ -0,0 +1,256 @@
+# hw definition file for processing by chibios_hwdef.py for YJUAV_A6Ultra
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# board ID for firmware load
+APJ_BOARD_ID 1144
+
+env OPTIMIZE -O2
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 5
+
+FLASH_SIZE_KB 2048
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 128
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART2 USART6 USART3 USART1 UART8 UART7 OTG2
+
+# now we define the pins that USB is connected on
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# USART2 TELEM1
+PD6 USART2_RX USART2
+PD5 USART2_TX USART2
+PD3 USART2_CTS USART2
+PD4 USART2_RTS USART2
+
+# USART6 TELEM2
+PC7 USART6_RX USART6
+PC6 USART6_TX USART6
+PG13 USART6_CTS USART6
+PG8 USART6_RTS USART6
+
+# USART3 GPS1
+PD9 USART3_RX USART3 NODMA
```

Comment:
I'm not sure you really need to switch off DMA on both of these ports - you don't have that many serial ports. But its your call

Reply:
This is a good suggestion, I will submit the override changes now, thank you.